.github/actions/build/action.yml

```
@@ -0,0 +1,22 @@
+name: 🪐 Build
+
+description: Build Everything
+
+on:
```

Can consider adding the paths-ignore and workflow_dispatch (which lets you trigger the build manually in github) - I've found that useful in the past.

```suggestion
on:
  push:
    branches:
      - main
    paths-ignore:
      - "**/*.md"
      - "**/*.yml"
      - "**/.gitignore"
      - "assets/*"

  # Allows you to run this workflow manually from the Actions tab
  workflow_dispatch:
```

---

Absolutely